documentation20/cn/08.connector/01.java/docs.md

```diff
@@ -600,6 +600,43 @@ public void setString(int columnIndex, ArrayList<String> list, int size) throws
 public void setNString(int columnIndex, ArrayList<String> list, int size) throws SQLException
 ```
 
+### <a class="anchor" id="schemaless_java"></a>无模式写入
+
+从 2.2.0.0 版本开始，TDengine 增加了对无模式写入功能。无模式写入兼容 InfluxDB 的 行协议（Line Protocol）、OpenTSDB 的 telnet 行协议和 OpenTSDB 的 JSON 格式协议。详情请参见[无模式写入](https://www.taosdata.com/docs/cn/v2.0/insert#schemaless)。
+注意：
+* JDBC-RESTful 实现并不提供无模式写入这种使用方式
+* 以下示例代码基于taos-jdbcdriver-2.0.36
+
+示例代码：
+```java
```

Comment:
need a blank line before code block

Reply:
add blank line as you see, but I wonder this is for beauty or format requirements?

Reply:
It's Markdown standard. You can use mdl to check it.